12_src/src/application.jsx

```diff
@@ -3,10 +3,10 @@
 // Now is the time to meet the first binding that redux-react (https://github.com/gaearon/react-redux)
 // brings to us: the Provider component.
 
-// Provider is a React Component designed to be used as a wrapper of your application's root component. Its
-// purpose is to provide your redux instance to all your application's components. How it does that does not
-// really matters to us but just to let you know, it's using React's context feature (it's undocumented so you
-// don't have to know about it but if you're curious:
+// Provider is a React Component designed to be used as a wrapper of your application's root component. It's
```

Comment:
Are you sure about this one? I really meant to use the possessive form, but maybe the sentence formulation is not clear... I meant "The purpose of Provider is to provide...".